ci/test_python.sh

```diff
@@ -108,7 +108,7 @@ echo "nx-cugraph coverage from networkx tests: $_coverage"
 echo $_coverage | awk '{ if ($NF == "0.0%") exit 1 }'
 # Ensure all algorithms were called by comparing covered lines to function lines.
 # Run our tests again (they're fast enough) to add their coverage, then create coverage.json
-pytest \
+NX_CUGRAPH_USE_COMPAT_GRAPHS=False pytest \
```

Comment:
This should work for both `True` and `False`, and it would be great to be able to test both options. `False` (not "compat" graph) matches how we were testing before.

Reply:
> it would be great to be able to test both options

Why can't we test both?  If we have to pick one, shouldn't we pick compat since (I'm assuming) it will be used much more?